Gemfile

```diff
@@ -9,10 +9,7 @@ gem 'jsbundling-rails'
 gem 'turbo-rails'
 gem 'propshaft', '0.4.0' # Changing this will break stuff. Work needed.
 
-gem 'mysql2',
-  github: 'brianmario/mysql2',
-  ref: '56c0ced2afe2cabe0b3f1a05e8e5e86713ebbc87'
-
+gem 'mysql2'
```

Comment:
Could you specify the specific version here pls (You can preceed with `~>` so that we get patch updates)